decred/decred/dcr/wire/msgblock.py

```diff
@@ -203,18 +195,17 @@ def btcEncode(self, pver):
         i += extraDataSize
         b[i] = ByteArray(self.stakeVersion, length=uint32).littleEndian()
         i += uint32
-        if i != MaxHeaderSize:
-            raise ValueError(f"unexpected BlockHeader encoded size {i}")
         return b
 
     def hash(self):
         """
         hash computes the block identifier hash for the given block header.
+
+        Encode the header and hash256 everything prior to the number of
+        transactions.  Ignore the error returns since there is no way the
+        encode could fail except being out of memory which would cause a
+        run-time panic.
         """
-        # Encode the header and hash256 everything prior to the number of
-        # transactions.  Ignore the error returns since there is no way the
-        # encode could fail except being out of memory which would cause a
-        # run-time panic.
```

Comment:
Can just nuke this. Translation doesn't really work.

Reply:
Nuking.